mix.exs

```diff
@@ -23,7 +23,8 @@ defmodule EdgeDB.MixProject do
       description: @description,
       package: package(),
       docs: docs(),
-      aliases: aliases()
+      aliases: aliases(),
+      compilers: [:yecc] ++ Mix.compilers()
```

Comment:
this shouldn't be here, the project doesn't contain any files for `yecc`. can you revert this please before merging?